content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -34,8 +34,13 @@ The operator is beta in 1.6 and suitable for production use.
     - A service to access operator metrics
     - Necessary Istio operator RBAC rules
 
-    See the available `istioctl operator init` flags to control which namespaces the controller and Istio are installed
-    into and the installed Istio image sources and versions.
+    You can also pass single or multiple namespaces to watch by using `--watchedNamespaces` flag.
+
+    {{< text bash >}}
+    $ istioctl operator init --watchedNamespaces=istio-namespace1,istio-namespace2
+    {{< /text >}}
+
+    See the available `istioctl operator init` flags to control which namespaces the operator controller and Istio are installed into and which namespaces to watch.
```

Comment:
Why are you removing the part: `and the installed Istio image sources and versions.`?

It seem this whole sentence could be improved. For example:
```suggestion
    You can configure which namespaces the operator controller and Istio are installed in, the namespace(s) the operator watches, the installed Istio image sources and versions, and more. See the [`istioctl operator init` command reference](/docs/reference/commands/istioctl/#istioctl-operator-init) for details.
```

Reply:
This looks good. Thanks